### ISSUE TITLE
Splat size is incorrect in orthographic camera view

### ISSUE THREAD

**Issue by hybridherbst** [aras-p/UnityGaussianSplatting#25]
When going into ortho mode to try out splat selection / splat editing, I noticed splat sizes are not correct there:

https://github.com/aras-p/UnityGaussianSplatting/assets/2693840/d2e3b108-4f26-4229-a502-1299beae8d31

**Comment by aras-p** [comment]
Yeah, the whole splat covariance matrix calculation goes wrong in ortho case, since that whole Jacobian matrix dance is very much assuming a perspective projection. And my math-fu is not strong enough to understand what *really* goes on there, i.e. how to make it also handle orthographic. Anyone know?
